example/drone32c.yaml

```diff
@@ -7,12 +7,12 @@ environment:
   - -5.5
   - -3
   - 0
-  obstacles:
-  - center: []
-    octomap_file: ../octomaps/drone32c.bt
-    octomap_stl: ../meshes/drone32c.stl
-    size: []
-    type: octomap
+  obstacles: []
+  # - center: []
+    # octomap_file: ../octomaps/drone32c.bt
+    # octomap_stl: ../meshes/drone32c.stl
+    # size: []
+    # type: octomap
```

Comment:
revert

Reply:
When  it was implemented static obstacles were not supported yet. I will add it.